### PR TITLE
Implementation of population frequencies …

### PR DESCRIPTION
**Pull request by savitakartik**
… with selection1d stream and fake frequency data

### CODE REVIEW

**Comment by savitakartik** [comment]
https://github.com/tskit-dev/tsqc/assets/25925005/7976a477-8177-465e-b034-70b59032ba46

**Comment by savitakartik** [comment]
There is a problem here, which I didn't foresee with the toy data I initially tried this on. We have to zoom in quite a bit to see individual mutation points that we can click on. So the index that the selection stream returns (which I'm using to locate the mutation the dataframe) corresponds to the row in the filtered mutations dataframe rather than the full muts_df. I'll have to fix this - perhaps by reusing the RangeStream we already have and subsetting the muts_df to the x and y ranges in the screen before fetching the population frequencies?

File: tsqc/pages/mutations.py [hunk 1]
```diff
@@ -102,6 +102,41 @@ def make_muts_panel(log_y, tsm):
...
+            ylim=(0, 20),
+        )
+        return bars
+
+    def update_mut_info_table(index):
+        if not index:
+            return hv.Table([], kdims=["info"], vdims=["value"]).opts(
+                width=config.PLOT_WIDTH, title="Tap on a mutation"
+            )
+        mut_data = muts_df.loc[index[0]].drop(pops)
```
Comment (savitakartik) [inline tsqc/pages/mutations.py:129]: I'm not sure how to avoid accessing the df row twice. I'm already doing the same thing in `update_pop_freq_plot`

**Comment by jeromekelleher** [comment]
LGTM. Can you make an issue to track removing the fake population data please, so we don't forget.

**Pull request merged**